docs/release-notes/release-2023-06-06.md

```diff
@@ -0,0 +1,150 @@
+---
+title: June 2023
+---
+
+# Release Notes June 2023 (2023-06-06)
+
+:::danger
+This release contains **breaking changes**
+:::
+
+## New Features
+
+- New decimal types
+- Support for documentation comments
```

Comment:
```suggestion
- Support for using doc comments instead of the @doc decorator
```

(I'm not even sure Documentation Comments are the same thing as doc comments).